documentation/reference/qfieldcloud/jobs.en.md

```diff
@@ -81,6 +81,17 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
 - The project is too big and the job has failed to run.
 - There are hidden files and directories within the project that are preventing the normal work of QFieldCloud. Hidden files and directories are those starting with a leading dot (`.`).
 
+### Re-apply changes in QFieldCloud
+
+To reapply the changes, in QFieldCloud go to to project in the Changes section, the changes are order in reverse order by the time of the changes, find the changes you want to re-apply and in the right side in the Action column click in the dropdown list.
```

Comment:
```suggestion
To reapply the changes, in QFieldCloud go to the project in the Changes section, the changes are ordered in reverse order by the time of the changes, find the changes you want to re-apply and on the right side in the Action column click on the dropdown list.
```

Reply:
Would be nice if we split this gigantic sentence into smaller digestable sentences.